core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -95,6 +97,8 @@
     private ElevationProvider eleProvider = ElevationProvider.NOOP;
     private FlagEncoderFactory flagEncoderFactory = FlagEncoderFactory.DEFAULT;
 
+    private BBox bounds = new BBox(-180, 180, -90, 90);
```

Comment:
This represents the BBox for the world-wide usecase. The inverse is not existing, right? There are test cases that do not call the init function, this is basically only for the test cases.